index.bs

```diff
@@ -4196,6 +4466,12 @@ To <dfn>generate a verbose debug report URL</dfn> given a [=verbose debug report
 1. Return the result of running [=generate a report URL=] with |report|'s
     [=verbose debug report/reporting origin=] and |path|.
 
+To <dfn>generate an aggregatable debug report URL</dfn> given an [=aggregatable debug report=] |report|:
+
+1. Let |path| be «"`debug`", "`report-aggregate-debug`"».
```

Comment:
Are we intentionally including `debug` twice in the path?

Reply:
I think it makes sense to be in `debug` path, and explicit `report-aggregate-debug` is to distinguish from `report-aggregate-attribution`.

Reply:
It would be disappointing to have this redundant URL forever, but we may be stuck at this point, alas.